e2e/serial/swap/1_swapFlow1.test.ts

```diff
@@ -117,6 +113,7 @@ it('should be able to connect to hardhat', async () => {
 });
 
 it('should be able to go to swap flow', async () => {
+  await delayTime('very-long');
```

Comment:
How certain are you about each of these delays?

Because we aren't fixing one particular issue with a single `delay` it's difficult to know which of these 'fixed' it.

I'm not saying we don't need all of these but it's something to think about because once these are merged they'll probably never be touched. We are usually in the business of adding delay, not checking to see if we can remove them.

Anyway, if any of these felt like a shot in the dark, I'd considering removing them and checking if they're necessary.

Reply:
Yeah, I was going through each test case by case for the most part and a lot of them were pretty necessary but i will def go back and check again on them. Its a good shout.

Reply:
ight ran these for a while and removed a lot of delays